platform/ios/src/MGLMapView.mm

```diff
@@ -5988,15 +5988,21 @@ - (void)updateUserLocationAnnotationViewAnimatedWithDuration:(NSTimeInterval)dur
 /// the overall map view (but respecting the content inset).
 - (CGPoint)userLocationAnnotationViewCenter
```

Comment:
If this PR will also introduce a way to also get and set the user location anchor point dynamically (such as `MGLMapView.userLocationAnchorPoint` & `[MGLMapView setUserLocationAnchorPoint]`), would I need to alter this method to accept an optional `CGPoint` argument  here so the method can use that argument?

Reply:
Oh...just kidding. I think I got confused. Instead of writing new methods to dynamically set the user location anchor point, we should just expose `updateUserLocationAnnotation` so that the developer can call this method to trigger the call to `userLocationAnnotationViewCenter` afterwards.